src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -4,9 +4,15 @@ import racingcar.strategy.NumberGeneratorStrategy
 
 class Car(
     point: Int = 0,
+    name: CarName,
 ) {
+    constructor(name: CarName) : this(0, name)
+    constructor(point: Int) : this(point, CarName(BLANK_NAME))
+    constructor() : this(0, CarName(BLANK_NAME))
```

Comment:
비슷한 구조가 많은거같아요!
```suggestion
class Car(
    point: Int = 0,
    name: CarName = CarName(BLANK_NAME),
) {
```
부생성자를 활용하기보단,
디폴트값을 설정하면 생성자를 추가로 선언하지 않아도 될거같아요!